### ISSUE TITLE
Show 'getting_started' hint if `machinery` is called without arguments

### ISSUE THREAD

**Issue by tgoettlicher** [SUSE/machinery#1286]
If you call `machiney help` the following hint is shown:
```
Machinery can show hints which guide through a typical workflow.
These hints can be switched off by '/usr/bin/machinery config hints=off'.

Hint: You can get started by inspecting a system. Run:
/usr/bin/machinery inspect HOSTNAME
```

The same hint should be shown if `machinery` is called without arguments.

**Comment by tgoettlicher** [comment]
This commit is related:
https://github.com/SUSE/machinery/commit/4b750e43ea33e8d0099efdd80fc707fbd6a03ee8

**Comment by keberle94** [comment]
When working on this I've seen that this change was on purpose to fix #369, so I think we should clarify first which way we want to go there.
